Dockerfile

```diff
@@ -1,3 +1,6 @@
 FROM yastdevel/ruby
 COPY . /usr/src/app
 
+# temporary to use the last packaging tasks
+# TODO: remove when it will be in openSUSE:Factory
```

Comment:
You do not have to wait for Factory, the Docker image installs the packages from YaST:Head: https://github.com/yast/docker-yast-ruby/blob/master/Dockerfile#L2 so the latest package should be installed soon... (You can force the Docker image to not wait for ~2hours auto rebuild).